quiz.sql

```diff
@@ -24,12 +37,12 @@ CREATE DATABASE westeros;
 
 -- Question 4. Change the motto of one of your houses. Look it up by its name.
 
--- your code starts here
+UPDATE houses
+SET motto = 'I love mottos'
+WHERE id = 1,2,3,4,5,6;
```

Comment:
You can't use commas like this in your SQL queries. If you wanted to select all of the houses, you could have used:
`UPDATE houses SET motto = 'I love mottos';`